examples/tv-casting-app/android/App/app/src/main/java/com/matter/casting/InitializationExample.java

```diff
@@ -53,6 +49,12 @@ public byte[] get() {
    * through commissioning
    */
   public static class CommissionableDataProvider implements DataProvider<CommissionableData> {
+    // Dummy values for commissioning demonstration only. These are hard coded in the example
+    // tv-app:
+    // connectedhomeip/examples/tv-app/tv-common/src/AppTv.cpp
+    private static final long DUMMY_SETUP_PASSCODE = 20202021;
+    private static final int DUMMY_DISCRIMINATOR = 3874;
```

Comment:
this should probably be randomly generated in a static initializer since there is no reason to have it hard coded.